easybuild/easyconfigs/o/OpenColorIO/OpenColorIO-1.1.0-foss-2018b.eb

```
@@ -0,0 +1,29 @@
+easyblock = 'CMakeMake'
+
+name = 'OpenColorIO'
+version = '1.1.0'
+
+homepage = 'http://opencolorio.org/'
+description = """ OpenColorIO (OCIO) is a complete color management solution geared towards motion picture production
+ with an emphasis on visual effects and computer animation."""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+
+source_urls = ['https://github.com/imageworks/OpenColorIO/archive/']
+sources = ['v%(version)s.tar.gz']
+patches = ['OpenColorIO-1.1.0_error.patch']
+checksums = [
+    '228589879e1f11e455a555304007748a8904057088319ebbf172d9384b93c079',  # v1.1.0.tar.gz
+    'f0ed3a99031870d2c0fce1be8595988dd9ad93de3ab311871ccb9d195fdefdfe',  # OpenColorIO-1.1.0_error.patch
+]
+
+builddependencies = [('CMake', '3.12.1')]
+
+separate_build_dir = True
+
+sanity_check_paths = {
+    'files': ['bin/ociobakelut', 'bin/ociocheck', 'lib/libOpenColorIO.so'],
```

@smoors Please use `SHLIB_EXT` rather than hardcoding `.so`:

```suggestion
    'files': ['bin/ociobakelut', 'bin/ociocheck', 'lib/libOpenColorIO.%s' % SHLIB_EXT],